### ISSUE TITLE
opcja::find_intrface()

### ISSUE THREAD

**Issue by ufsowa** [ufsowa/mcdif#17]
Add function of opcja::find_intrface()
to find position of interface

**Comment by ufsowa** [comment]
Base on concentration in phases.

Calculate total C in whole sample Cavg.
Calculate C in rezervuars Crez.
Calculate atomic % C in region and compare with Crez.
:facepunch:

**Comment by ufsowa** [comment]
Still wrong. `do_equi` Must distinguish phases.
Major component in region must be used.  

First: find phase -> will give direction
Second: choose type of atom due to proportion in phase
Third: If Phase can not be determined, look at fluxex -> will give a direction
    Choose type of atom due to proportion in phase indicated be direction

:facepunch: 

This simulate two things:
- Vacancy creation/anihilation
- lattice (mechanical) relaxation

Vacancy change is due to off-stoichiometry
Lattice relaxation, place where atom will appear/disappear due to vacancy creation/anihilation, must be due to direction of relaxation.

This direction is determined be strain vector in lattice. Moreover, every phase has their own strain vector Fs.

Fs is proportional to vacancy flux in the phase.

So not local strain vector fs is in interest but total in the phase Fs.

Questions:
- How to distinguish between phases?
- Does sum of Fs over all phases in the material should go to zero?
  In other words, Should Fs be opposite to each other?

**Comment by ufsowa** [comment]
Second question refers to matano Plane as a criterium.
But maano plane in general, is not the same as a interface.
For example, when flux of vacancies is assymetric.

So, I must stay with intrface criteria asa a concentration and use majority criterium, as a definition of phase.

But matano is defined for atoms.

In case of vacancies, this is real criterium.

**Comment by ufsowa** [comment]
First of all: make a test with majority component. Majority is wrong.
But prportional to phase concentration is ok. :facepunch:
